resources/i18n/pt_BR/cura.po

```diff
@@ -4051,28 +3913,27 @@ msgstr "Biblioteca de recorte de polígonos"
 #: /home/trin/Gedeeld/Projects/Cura/resources/qml/Dialogs/AboutDialog.qml:155
 msgctxt "@label"
 msgid "JSON parser"
-msgstr ""
+msgstr "parser JSON"
```

Comment:
We're going to add a capital letter here. You couldn't have known that from the original text, but it's consistent if it would have capital letters.